openvalidators/neuron.py

```diff
@@ -130,23 +130,52 @@ def __init__(self):
         bt.logging.debug("loading", "reward_functions")
         if self.config.neuron.mock_reward_models:
             self.reward_functions = []
+            self.reward_weights = []
             self.masking_functions = [
-                MockRewardModel('blacklist_filter'), 
-                MockRewardModel('nsfw_filter') 
+                MockRewardModel(RewardModelType.blacklist.value),
+                MockRewardModel(RewardModelType.nsfw.value),
```

Comment:
Why is `nsfw` a masking function rather than a reward function?

Reply:
I think @Eugene-hu and @isabella618033 will know to answer this better than me. From what I understood, they are played out like a binary filter, where it either returns 0 or 1, but not sure if that's the case.

Reply:
We actually have a NSFW threshold parameter `bound` which maps the float value to 0 or 1. Would be good to have the raw values in there for deeper introspection about flagged completions

Reply:
I like this idea, I think we can create a new issue to map this feature